src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (...args) => {
+    const returnValue = sum;
+
+    if (!args.length) {
+      sum = 0;
+
+      return returnValue;
+    }
```

Comment:
I would move the variable  under if block, 'cause it's being used only there
```suggestion
  const adder = (...args) => {
    if (!args.length) {
      const returnValue = sum;
      sum = 0;

      return returnValue;
    }
```